src/main/java/org/opensearch/security/auditlog/sink/Log4JSink.java

```diff
@@ -27,7 +27,7 @@ public final class Log4JSink extends AuditLogSink {
 
     public Log4JSink(final String name, final Settings settings, final String settingsPrefix, AuditLogSink fallbackSink) {
         super(name, settings, settingsPrefix, fallbackSink);
-        loggerName = settings.get(settingsPrefix + ".log4j.logger_name", "sgaudit");
+        loggerName = settings.get(settingsPrefix + ".log4j.logger_name", "audit");
```

Comment:
Let's add a deprecation check on the old value and OR the setting result. 

While it's a bug that the value is incorrect, it would be a breaking change for 2.x customers

Reply:
@peternied The key for the setting does not change, its the default value that changes.

In order to not use the default value, an administrator can supply the setting in `opensearch.yml` like this:

```
plugins.security.audit.config.log4j.logger_name: customauditlogger
```

Reply:
I think I misunderstand the bug, what is the old vs new behavior with this change?

Reply:
Its the name of the log4j logger. When you enable log4j, the audit logs are included in the OpenSearch logs like this:

```
[2024-03-21T10:17:00,802][INFO ][sgaudit                  ] [smoketestnode] {"audit_cluster_name":"opensearch","audit_node_name":"smoketestnode","audit_trace_task_id":"vBWwR9s9TwyJbO5MPu6ASA:100","audit_transport_request_type":"MainRequest","audit_category":"GRANTED_PRIVILEGES","audit_request_origin":"REST","audit_node_id":"vBWwR9s9TwyJbO5MPu6ASA","audit_request_layer":"TRANSPORT","@timestamp":"2024-03-21T14:17:00.797+00:00","audit_format_version":4,"audit_request_remote_address":"::1","audit_request_privilege":"cluster:monitor/main","audit_node_host_address":"10.18.199.16","audit_request_effective_user":"admin","audit_node_host_name":"10.18.199.16"
```

The logger name is the section that appears after the log level and before the node name.